sdk/core/Azure.Core/src/GeoJson/GeoJsonConverter.cs

```diff
@@ -127,6 +127,11 @@ internal static GeoObject Read(JsonElement element)
 
             if (element.TryGetProperty(BBoxProperty, out JsonElement bboxElement))
             {
+                if (bboxElement.ValueKind == JsonValueKind.Null)
```

Comment:
Non blocking curiosity question - is null a valid value according to the schema?  If so 👍🏻.  If not, an alternative is to JsonException.

Reply:
I just looked at the spec, and you're right it looks like null is not valid [according to the schema](https://www.rfc-editor.org/rfc/rfc7946.html#section-5):
> A GeoJSON object MAY have a member named "bbox" to include
   information on the coordinate range for its Geometries, Features, or
   FeatureCollections.  The value of the bbox member MUST be an array of
   length 2*n where...

The error is already throwing an informative JsonException so I think this PR & the issue can be closed:
```
System.Text.Json.JsonException : The JSON value could not be converted to Azure.Core.GeoJson.GeoObject. Path: $ | LineNumber: 8 | BytePositionInLine: 1.
---- System.InvalidOperationException : The requested operation requires an element of type 'Array', but the target element has type 'Null'.
```

Thank you @annelo-msft for calling this out! And sorry for not having checked this before!

Reply:
... which begs the question of "why did Pavel make the property nullable, then?"

Reply:
@jsquire I think because the Bounding Box isn't required, so if one isn't provided then it would be null in that case. But if there is a "bbox" in the JSON it has to be an array

Reply:
ah, that's the nuance that I was missing.   So, the behavior was entirely correct.   It's optional but not nullable in the JSON.